examples/test/models/cpp/TestInterfaceCppImplementation.h

```diff
@@ -104,6 +104,13 @@ class TestInterfaceCppImplementation : public TestInterfacePropertyAdapter
     {
     }
 
+    void asyncMethod(int arg, facelift::AsyncAnswer<int> answer) override
+    {
+        QTimer::singleShot(1000, [answer, arg]() mutable {
+            answer(arg + 10);
+        });
+    }
```

Comment:
It would be good to show the usage (TestUI) as well.

Reply:
Example available in MyApp.qml